homeassistant/components/reolink/const.py

```diff
@@ -0,0 +1,13 @@
+"""Constants for the Reolink Camera integration."""
+
+DOMAIN = "reolink"
+PLATFORMS = ["camera"]
```

Comment:
`PLATFORMS` is only used in the init module. We can move it there. Please move the constants that are only used in a single module to that module. That makes it easier to read each module.